introduction.md

```diff
@@ -1,3 +1,5 @@
 Hello friends
 codeschool/courses/python
 code school path: http://campus.codeschool.com/courses/Ruby
+* Git
+* python
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?